app/modules/site_settings/helpers.py

```diff
@@ -143,7 +143,7 @@ def _validate_fields(cls, value, class_name):
         ]
         schema_fields = [
             ('category', str, True),
-            ('description', str, False),
+            # ('description', str, False),  # think this is *optional*
```

Comment:
This should be removed, setting False makes it optional, I'll do that later

Reply:
perfect, i was just not 100% sure it should go.

Reply:
@hwindsor this is the line i mention in slack.  this seems to be _back_ in the final commit.  not sure what is going on here.